CONTRIBUTING.markdown

```diff
@@ -1,114 +1,171 @@
-*Java* &rarr; *Code Style* &rarr; *Formatter* &rarr; *Active Profile* menu.
-* Include a reference to your ticket (e.g. LPS-XXXXX) in your commit messages.
-For example:
-
-        LPS-83432 Make the example in CONTRIBUTING imperative and concrete
-
-* *Test* your changes thoroughly! Consider the wide variety of operating
-systems, databases, application servers, and other related technologies Liferay
-supports. Make sure your changes in one environment don't break something in
-another environment. See [Unit and Integration
-Tests](http://www.liferay.com/community/wiki/-/wiki/Main/Unit+and+Integration+tests)
-for details on executing Liferay's automated tests.
-
-## Submitting Changes
-
-* Push changes in your branch to your fork.
-* Send a pull request to the core maintainer of the area to which your changes
-apply. You can use [github.com](https://github.com/) to initiate the pull
-request, or you can use
-[git-pull-request](https://github.com/liferay/git-tools/tree/master/git-pull-request),
-a command line tool for issuing pull requests, developed specifically for the
-Liferay Community.
-* In the LPS ticket, provide a link to your GitHub pull request and respond to
-the [Contributor License
-Agreement](http://www.liferay.com/legal/contributors-agreement) displayed by
-clicking the *Contribute Solution* button.
-* You're done! Well, not quite ... be sure to respond to comments and questions
-to your pull request until it is closed.
-
-## Additional Resources
-
-* [Getting Started as a Liferay
-Developer](http://www.liferay.com/community/wiki/-/wiki/Main/Getting+started+as+a+Liferay+Developer+in+a+few+steps)
-* [Optimal Liferay Core
-Development](http://www.liferay.com/community/wiki/-/wiki/Main/Optimal+Liferay+Core+Development)
+Liferay is developed by its community consisting of Liferay users, enthusiasts, employees, customers, partners, and others. We strongly encourage you to contribute to Liferay's open source projects by implementing new features, enhancing existing features, and fixing bugs. We also welcome your participation in our forums, writing documentation, and translating existing documentation.
+
+Liferay is known for its innovative top quality features. To maintain this reputation, all code changes are reviewed by a core set of Liferay project maintainers. We encourage you to introduce yourself to the core maintainer(s) and engage them as you contribute to the areas they maintain.
+
+To get a deeper understanding of Liferay in general, make sure to read Liferay’s official documentation on the [Liferay Developer Network](http://dev.liferay.com), for [using](https://dev.liferay.com/discover/portal) and [deploying](https://dev.liferay.com/discover/deployment) Liferay Portal, and [developing](https://dev.liferay.com/develop/tutorials) on Liferay. This documentation contains extensive explanations, examples, and reference material for you to consult time and time again.
```

Comment:
Before DevCon we will be re-branding the Liferay Developer Network as the new Liferay Portal website. As part of this we will also gather all the documentation under one single [Documentation](dev-uat.liferay.com/documentation) section. You can see a preview at DEV-UAT. 

We should probably adapt this text taking those upcoming changes into consideration.